cluster/common.sh

```diff
@@ -325,7 +325,7 @@ function find-release-tars() {
   # This tarball is used by GCI, Ubuntu Trusty, and Container Linux.
   KUBE_MANIFESTS_TAR=
   if [[ "${MASTER_OS_DISTRIBUTION:-}" == "trusty" || "${MASTER_OS_DISTRIBUTION:-}" == "gci" || "${MASTER_OS_DISTRIBUTION:-}" == "ubuntu" ]] || \
```

Comment:
do we need to allow this for master os distribution as well, or do those tests still use GCI for the master?

Reply:
All tests use cos as master image. BTW, we should probably also remove ubuntu from master os some time later.

Reply:
Ok, great.